tools/internal_ci/windows/grpc_run_tests_matrix.bat

```diff
@@ -17,6 +17,11 @@ cd /d %~dp0\..\..\..
 
 call tools/internal_ci/helper_scripts/prepare_build_windows.bat || exit /b 1
 
+@rem Only install Python interpreters if we are running Python tests
```

Comment:
shouldn't this be in prepare_build_windows.bat? (it's exatly the same idea as prepare_build_macos_rc: https://github.com/grpc/grpc/blob/e36246bc9d65bc241f0b26671590b2b25f10df72/tools/internal_ci/helper_scripts/prepare_build_macos_rc#L85)

Reply:
Good idea. Changed to using env var `PREPARE_BUILD_INSTALL_DEPS_PYTHON` to configure the install.